_compdemos/rslurm_intro.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Introduction to rslurm
+date: 2020-02-27
+main_author: Michael Gutteridge
+primary_reviewers: vortexing, atombaby, jennylsmith
+---
+
+The [`rslurm`](https://cran.r-project.org/web/packages/rslurm/index.html) library creates a mechanism for distributing R calcuations across a set of Slurm jobs.  `rslurm` creates and manages self-contained Slurm jobs, then combining the output from the various jobs as they complete.
+
+## Purpose
+
+- Learn basic use of `rslurm`
+- Create a script for calculating &pi; using the [Monte Carlo method](https://academo.org/demos/estimating-pi-monte-carlo/)
+
+## Audience
+
+You should know:
+
+- How to use `sbatch` to submit jobs to a Slurm cluster
+- Some R programming experience
+
+## Using rslurm
+
+### Requirements
+
+You will need access to the Rhinos for submitting the job 
+
+### Instructions on how to get set-up
+
+Log into a rhino or in an NoMachine terminal session.  Use the module commands to load R- most later versions of R in our environment have the `rslurm` module ainstalled but it is also available for installation via CRAN
```

Comment:
Could add a link to NoMachine (https://sciwiki.fredhutch.org/scicomputing/access_methods/#nomachine-nx-multi-os). If the reader hasn't heard of it, they may be interested in learning more.